client/lib/post-normalizer/rule-create-better-excerpt.js

```diff
@@ -90,8 +93,11 @@ export default function createBetterExcerpt( post ) {
 		return post;
 	}
 
-	post.better_excerpt = formatExcerpt( post.content );
-	post.better_excerpt_no_html = stripHTML( post.better_excerpt );
+	const strippedDom = buildStrippedDom( post.content );
+	post.content_no_html = striptags( strippedDom );
```

Comment:
no, strippedDom has some things we don't want to count removed from the dom.

striptags with no args is effectively stripHTML, I think.